### PR TITLE
chore(ssa refactor): hookup alloc + store instructions

### PR DESCRIPTION
**Pull request by joss-aztec**
# Description

## Problem\*

Resolves 

## Summary\*

This PR sets out to

### Example

Before:

```

```

After:

```

```

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs [hunk 1]
```diff
@@ -143,11 +143,12 @@ impl Context {
                 let result_ids = dfg.instruction_results(instruction_id);
                 if Self::value_is_array_address(result_ids[0], dfg) {
                     self.track_array_address(result_ids[0], binary, dfg);
-                    return;
+                    (Vec::new(), Vec::new())
+                } else {
+                    let result_acir_var = self.convert_ssa_binary(binary, dfg);
+                    self.ssa_value_to_acir_var.insert(result_ids[0], result_acir_var);
```
Comment (kevaundray) [inline crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs:149]: Is this extra line needed?

Comment (kevaundray) [reply crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs:149]: Shouldn't it do that when we return the vectors?

**Review by kevaundray — COMMENTED** [review]
LGTM -- one comment

File: crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs (hunk 1 above)
Comment (joss-aztec) [reply crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs:149]: Good spot. Removed

Comment (joss-aztec) [reply crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs:149]: Oh, wait, bad fix

Comment (joss-aztec) [reply crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs]: Reverted, and deleted the correct line this time

**Review by kevaundray — APPROVED** [review]

**Pull request merged**